xep-0319.xml

```diff
@@ -25,6 +25,12 @@
   <shortname>idle</shortname>
   &tobias;
   <revision>
+    <version>1.0.1</version>
+    <date>2017-05-30</date>
+    <initials>egp</initials>
+    <remark><p>Add a precision about the &xep0082; profile used.</p></remark>
```

Comment:
Double-nit. You can have "A precision", e.g. "This is accurate to a precision of one Cochrane." ;)